<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background : rgb(255, 255, 255)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Filename    : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>511</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter the path directly or select it from the cells below...</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>711</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="fileInput1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>511</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);border-width:1px;border-style:solid;

</string>
    </property>
    <property name="text">
     <string>MFT path...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fileSelect">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>90</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(147, 210, 232)</string>
    </property>
    <property name="text">
     <string>fileSelect</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>MFT path     : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="analyze_btn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>540</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(147, 210, 232)</string>
    </property>
    <property name="text">
     <string>Analyze</string>
    </property>
   </widget>
   <widget class="QPushButton" name="analyze_btn_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>540</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>291</width>
      <height>18</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="handleWidth">
     <number>37</number>
    </property>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Cloud    : </string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Dropbox</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GoogleDrive</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NaverWorks</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="enter">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>120</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(147, 210, 232)</string>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Result : </string>
    </property>
   </widget>
   <widget class="QLabel" name="anlysis_result">
    <property name="geometry">
     <rect>
      <x>113</x>
      <y>500</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_line_edit">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>150</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>150</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Filter</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>711</width>
      <height>22</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="opaqueResize">
     <bool>true</bool>
    </property>
    <property name="handleWidth">
     <number>20</number>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radio_MFT">
        <property name="text">
         <string>$MFT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_MFTcsv">
        <property name="text">
         <string>MFT csv</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="mftdescriptor">
     <property name="text">
      <string>Description..</string>
     </property>
    </widget>
   </widget>
   <zorder>splitter_3</zorder>
   <zorder>label_2</zorder>
   <zorder>lineEdit</zorder>
   <zorder>tableWidget</zorder>
   <zorder>fileInput1</zorder>
   <zorder>fileSelect</zorder>
   <zorder>label_3</zorder>
   <zorder>analyze_btn</zorder>
   <zorder>analyze_btn_2</zorder>
   <zorder>splitter</zorder>
   <zorder>enter</zorder>
   <zorder>label_4</zorder>
   <zorder>anlysis_result</zorder>
   <zorder>label_5</zorder>
   <zorder>search_line_edit</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>776</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
